scripts/bootstrap.sh.in

```diff
@@ -144,56 +144,77 @@ retry_dns_count: 3
 EOF
 }
 
-set_salt_command() {
-    echo "Setting salt master command"
-    # shellcheck disable=SC2046
-    SALT_MASTER_CALL="crictl exec -i $(crictl ps -q --label io.kubernetes.container.name=salt-master --label io.kubernetes.pod.namespace=kube-system)"
+get_salt_container() {
+    local salt_container
+    salt_container=''
+    while [ -z "$salt_container" ]; do
```

Comment:
This could loop forever, needs a break/failure at some point.

Reply:
Good point, will do.